.changeset/sixty-years-exist.md

```diff
@@ -0,0 +1,5 @@
+---
+'@microsoft/atlas-css': minor
+---
+
+Add standardized message chat styles
```

Comment:
`Adding message component."

We need another changeset for site:

`Adding documentation page for message component.`

Reply:
updated

Reply:
I think what Olga is suggesting is to have a separate changeset file for the site so it can have its own changeset message `Adding documentation page for message component.`.

Reply:
@jim-benson What Ming says is correct - we normally have separate changesets for each area, so please update and ensure the description says `Adding message component` for the `atlas-css` changes, and `Adding message component` for the chat to match what we've done for others.

Here are a few changelogs files so you can see the structure:
- https://github.com/microsoft/atlas-design/blob/main/css/CHANGELOG.md
- https://github.com/microsoft/atlas-design/blob/main/site/CHANGELOG.md

Reply:
updated